### PR TITLE
Daily Rounds: Updated consciousness levels choices and migrated `CANNOT BE ASSESSED` to `UNKNOWN`

### PR DESCRIPTION
**Pull request by cp-Coder**
## Proposed Changes

- [x] Updates Consciousness Levels Choices
- [x] Custom Migration: Migrate old option `CANNOT_BE_ASSESSED` to `UNKNOWN`

### Updated Choices: 
```
UNKNOWN (untouched)
ALERT (untouched)
RESPONDS_TO_VOICE (renamed)
RESPONDS_TO_PAIN (renamed)
UNRESPONSIVE (mapped to UNKNOWN)
AGITATED_OR_CONFUSED (new)
ONSET_OF_AGITATION_AND_CONFUSION (new)
```

## Associated Issue

- Issue: https://github.com/coronasafe/care_fe/issues/6434
- FE PR: https://github.com/coronasafe/care_fe/pull/6435

### Before Migration (Map CANNOT BE ASSESSED - 25 to UNKNOWN - 0)
image

### After Migration (Map CANNOT BE ASSESSED - 25 to UNKNOWN - 0)
image

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [x] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

File: care/facility/migrations/0390_alter_dailyround_consciousness_level.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    records = daily_round_model.objects.all()
+
+    entries = []
+
+    for record in records:
+        consciousness_level = record.consciousness_level
+        if consciousness_level == 25:
+            record.consciousness_level = 0
+            entries.append(record)
+    daily_round_model.objects.bulk_update(entries, ["consciousness_level"])
```
Comment (rithviknishad) [inline care/facility/migrations/0390_alter_dailyround_consciousness_level.py:17]: Consider optimizing the query

```suggestion
    qs = daily_round_model.objects.filter(consciousness_level=25)
    qs.update(consciousness_level=10)
```

**Review by rithviknishad — APPROVED** [review]
Rest LGTM

**Comment by nihal467** [comment]
bug in the front-end, but functionality is partially ok

**Review by Ashesh3 — APPROVED** [review]

**Review by sainak — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM @vigneshhari can you review it

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**